library/ws-widget/collection/src/lib/_common/content-toc/app-toc-content-card-v2/app-toc-content-card-v2.component.html

```diff
@@ -148,6 +150,11 @@
                             content.primaryCategory === primaryCategory.COMP_ASSESSMENT ||
                             content.primaryCategory === primaryCategory.STANDALONE_ASSESSMENT ||
                             content.primaryCategory === primaryCategory.PRACTICE_RESOURCE">
+                            <div style="width:300px;">
+                                <span style="display: inline;">    {{  content | json  }} ========== </span>
```

Comment:
this code should get removed, please confirm, on UI we are not showing json anywhere